ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/Block.java

```diff
@@ -63,6 +63,7 @@ public void writeTo(final RLPOutput out) {
     out.writeList(body.getOmmers(), BlockHeader::writeTo);
     body.getWithdrawals().ifPresent(withdrawals -> out.writeList(withdrawals, Withdrawal::writeTo));
     body.getDeposits().ifPresent(deposits -> out.writeList(deposits, Deposit::writeTo));
+    body.getExits().ifPresent(exits -> out.writeList(exits, ValidatorExit::writeTo));
```

Comment:
We could potentially have `body.writeTo(out)` here instead of reimplementing the logic. Out of the scope of this PR though.